File: easy/even_numbers.py
#!/usr/bin/env python2
"""
- CodeEval: Even Numbers
- https://www.codeeval.com/open_challenges/100/
"""

from sys import argv

def main():
    with open(argv[1]) as fh:
        for line in fh:
            number = int(line)
            modulo = number % 2
            if modulo:
                print(0)
            else:
                print(1)


if __name__ == "__main__":
    main()

File: easy/even_numbers.c
/* Code Eval - Even Numbers
 * https://www.codeeval.com/open_challenges/100/
 */

#include <stdio.h>

int main(int argc, const char * argv[]) {
    FILE *file = fopen(argv[1], "r");
    char line[1024];
    while (fgets(line, 1024, file)) {
        int number = atoi(line);
        printf("%d\n", !(number%2));
    }
    return 0;
}
